helpers/main_helper.py

```diff
@@ -0,0 +1,75 @@
+import random, string, itertools
+
+# English words searched in a grid
+words = open('words.txt', 'r').read().splitlines()
+# All low case letters from English alphabet
+letters = string.ascii_lowercase
+found_words = set()
+
+
+def create_grid(a, b):
+	"""Create grid of letters with a x b dimensions"""
+	if a < 1 or b < 1: raise Exception('Dimensions of grid can\'t be less than 1')
+	global grid_letters
+	grid_letters = [list(map(lambda let: random.choice(letters), range(a))) for _ in itertools.repeat(object, b)]
+
+
+def grid_transpose(b):
+	"""Transpose grid"""
+	global transpose_letters
+	transpose_letters = [row[i] for i in range(len(grid_letters[0])) for row in grid_letters]
+	transpose_letters = [transpose_letters[x:x + b] for x in range(0, len(transpose_letters), b)]
+
+
+def grid_diagonals():
```

Comment:
what is the computational complexity of this algorithm?  (i.e. how does the width and height of the grid affect the runtime)

Reply:
transpose_letters is a width x height or all elements of grid and `b` is a height or length of grid_letters array

Reply:
I've been working on this project for a day in total and wanted to create this diagonals algorithm myself, so here is what I've done: (screenshots attached)

it works for any size: 4x4 or 4x2 or 2x4 

first `for` loop -> all diagonals from left to right(starts with first element) then up
<img width="409" alt="diagonals_primary_up" src="https://user-images.githubusercontent.com/50090715/56926656-3e0f1c00-6aa0-11e9-9e29-9dd41a37f83b.png">
second `for` loop -> all diagonals from right to left(starts with last element) then up
<img width="409" alt="diagonals_secondary_up" src="https://user-images.githubusercontent.com/50090715/56926658-3e0f1c00-6aa0-11e9-99b0-dee4ac92e780.png">
third `for` loop -> all diagonals for from to right(starts with second row first element) then down
<img width="410" alt="diagonals_primary_down" src="https://user-images.githubusercontent.com/50090715/56926655-3e0f1c00-6aa0-11e9-993e-61dfa050d6e1.png">
fourth `for` loop -> all diagonals for from to right(starts with second row last element) then down
<img width="409" alt="diagonals_secondary_down" src="https://user-images.githubusercontent.com/50090715/56926657-3e0f1c00-6aa0-11e9-84c0-ed90fef4ae70.png">